docroot/themes/humsci/humsci_basic/src/scss/utilities/_fonts.scss

```diff
@@ -70,3 +70,35 @@ a.hb-pill {
     @include hb-blockquote;
   }
 }
+
+.hb-title {
```

Comment:
Can we put a note in the H&S Patterns docs that states this class is only intended to be used in layout builder? I tried to apply it to a view and I couldn't get it to work correctly. Even if it did, looking at how the font-size is 3.8rem, I don't think we want it to work in a view. :eyes: